Frontend/app/common/components/error-summary/macro.njk

```diff
@@ -0,0 +1,11 @@
+{% from 'components/error-summary/macro.njk' import errorSummary as nhsErrorSummary %}
```

Comment:
are we going to use this?

Reply:
if yes we can move to BCL

Reply:
yes we are 😒.  OK to do it after this PR?